geomesa-blobstore/geomesa-blobstore-core/src/main/scala/org/locationtech/geomesa/blob/core/AccumuloBlobStore.scala

```diff
@@ -87,9 +114,9 @@ class AccumuloBlobStore(ds: AccumuloDataStore) {
     (value.get, filename)
   }
 
-  private def putInternal(file: File, id: String) {
-    val localName = file.getName
-    val bytes =  ByteStreams.toByteArray(Files.newInputStreamSupplier(file))
+  private def putInternal(file: File, id: String, params: Map[String, String]) {
+    val localName = getFileName(file, params)
+    val bytes = ByteStreams.toByteArray(Files.asByteSource(file).openBufferedStream())
```

Comment:
This probably doesn't close the input stream and makes 2 copies of the byte array...plus this is simpler:
`val bytes = Files.toByteArray(file)`